zzz_generated/app-common/app-common/zap-generated/af-structs.h

```diff
@@ -23,6 +23,8 @@
 #include <stdint.h>
 
 #include <app/util/basic-types.h>
+#include <lib/core/GroupId.h>
```

Comment:
Are these generated? The name suggests so in which case zap sources need to be changed.

My bet is on src/app/zap-templates/templates/app/af-structs.zapt

Reply:
This will likely fail the ZAP gen CI check. You need to update af-structs.zapt and then regen-all (run `./scripts/tools/zap_regen_all.py`)

Reply:
Tried rerun zap and got an `FileNotFoundError: [Errno 2] No such file or directory: 'node'` error.

Rebased off master and somehow CI not running still

Reply:
I ran zap regen and regenerated things.

CI is indeed not running and I do not know why :(

Reply:
CI started after my push after all. yay!